00-HTML-CSS-basics/Ex02/css/style.css

```diff
@@ -0,0 +1,42 @@
+html {
+	font-family:'Amatica SC', Times, serif;
+	font-size: 14px;
+}
+header {
```

Comment:
It's fine, just check that all the style related to html tags could be added inside the class related to each one.